src/lib/core/CHIPConfig.h

```diff
@@ -1169,6 +1169,19 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_ENABLE_SERVER_IM_EVENT 1
 #endif
 
+/**
+ * Accepts receipt of invalid privacy flag usage that affected some early SVE2 test event implementations.
+ * When SVE2 started, group messages would be sent with the privacy flag enabled, but without privacy encrypting the message header.
+ * The issue was subsequently corrected in master, the 1.0 branch, and the SVE2 branch.
+ * This is a temporary workaround for interoperability with those erroneous early-SVE2 implementations.
+ * The cost of this compatibity mode is twice as many decryption steps per received group message.
+ *
+ * TODO: Remove this workaround once interoperability with legacy pre-SVE2 is no longer required.
```

Comment:
File a tracking issue, perhaps xref in the comment?

Reply:
Will add ref to #24573